src/main/java/org/gradle/profiler/studio/launcher/StudioLauncher.java

```diff
@@ -64,6 +71,24 @@ private void logLauncherConfiguration(List<String> commandLine) {
         System.out.printf("* Using command line: %s%n%n", String.join(" ", commandLine));
     }
 
+    private Map<String, String> writeIdeaProperties() {
+        if (!studioSandbox.getConfigDir().isPresent()) {
```

Comment:
:x: I think we can just create this file also in the`studioSandbox.getJvmArgsDir()` so we can set `idea.properties` even if config dir is not present. `jvmArgsDir` is just some temp folder that we currently use only for `idea.vmoptions`.

:thinking: We could then maybe also merge `writeAdditionalJvmArgs` and `writeIdeaProperties` methods.

:nail_care: Maybe it would be worth also renaming `jvmArgsDir` to something else.